pubspec.yaml

```diff
@@ -4,26 +4,26 @@ description: Easy and Fast internationalizing and localization your Flutter Apps
 homepage: https://github.com/aissat/easy_localization
 issue_tracker: https://github.com/aissat/easy_localization/issues
 
-version: 2.3.1-dev.2
+version: 3.0.0-dev.0
 
 environment:
-  sdk: ">=2.6.0 <3.0.0"
+  sdk: ">=2.7.0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  intl: ^0.16.0
-  shared_preferences: ^0.5.7
-  args: ^1.5.2
-  path: ^1.6.4
+  intl: ">=0.16.0 <=0.17.0-nullsafety.2"
+  shared_preferences: ^0.5.12+4
```

Comment:
">=0.5.12+4 <2.0.0"

https://pub.dev/packages/shared_preferences#backward-compatible-100-version-is-coming

Reply:
can't  `0.16.0 `use  coz f`lutter_localizations `from sdk depends on `intl 0.17.0-nullsafety.2 `

```log
Because every version of flutter_localizations from sdk depends on intl 0.17.0-nullsafety.2 and easy_localization depends on intl ^0.16.1, flutter_localizations from sdk is forbidden.
```

Reply:
in stable channel?

Reply:
`Channel dev, 1.26.0-1.0.pre`

Reply:
#313